roadrunner.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+# Namespace for all RoadRunner code
+module RoadRunner
```

Comment:
I would think you'd want more logic in here. The tasks should be pretty light weight. If benchmark file exists, convert with this module, and the module should probably return the full file's path. The task currently has conversion logic where it probably shouldn't.